src/misc/bonemeal.ts

```diff
@@ -0,0 +1,26 @@
+import { List } from 'java.util';
+import { Material } from 'org.bukkit';
+import { BlockState } from 'org.bukkit.block';
+import { BlockFertilizeEvent } from 'org.bukkit.event.block';
+
+registerEvent(BlockFertilizeEvent, (event) => {
+  const blocks: List<BlockState> = event.getBlocks();
+  const cancel = !canBeFertilized(blocks);
+  event.setCancelled(cancel);
```

Comment:
```suggestion
  event.cancelled = cancel;
```
Explained in #13 review comment.